serverless.yml

```diff
@@ -14,6 +14,8 @@ provider:
         path: ./ingest-zip/
       image:
         path: ./ingest-image/
+      delete:
+        path: ./ingest-delete/
```

Comment:
The Severless Framework [docs](https://www.serverless.com/framework/docs/providers/aws/guide/functions?prod_DOCS_SEARCH%5Bquery%5D=batch#referencing-container-image-as-a-target) seem to suggest that:

> Serverless will create an ECR repository for your image, but it currently does not manage updates to it. An ECR repository is created only for new services or the first time that a function configured with an image is deployed. 

I just want to make sure I understand the implications here. Does that mean that images won't get re-built and pushed to ECR if we make updates to the Dockerfiles, package.json, and handler code? Or just some of those files?

Reply:
@nathanielrindlaub This is related to why we manage all the Dockerfiles as "lambdas" even if they are going to be run on batch. Serverless handles all the updates as one expects and then we can just ignore the quirks of "batch only" images.